Jenkinsfile

```diff
@@ -0,0 +1,18 @@
+pipeline {
+    agent any
+
+    stages {
+        stage('Build') {
+            steps {
+                sh 'mkdir packages && cd packages && nuget install ../Integreat/Integreat.Droid/packages.config'
+                sh 'msbuild Integreat.sln /p:AndroidSdkDirectory="/opt/android-sdk/" || true'
+                sh 'msbuild Integreat/Integreat.Droid/Integreat.Droid.csproj /p:AndroidSdkDirectory="/opt/android-sdk/" /target:SignAndroidPackage'
```

Comment:
SignAndroidPackage target is not available in Localisation project so we only build the Droid project.